### DISCUSSION TITLE
Transmission keeps complaining about libtorrent

### DISCUSSION THREAD

**Discussion by diddiman** [Flexget/Flexget#3213] [Q&A] [answered]
Hello

I have this job that is basically trying to download stuff through transmission. It keeps saying _libtorrent package required_
Does anyone have an idea how to resolve this?

I am using the latest version of flexget, 3.7.3 python. I use it in venv, and i'm on a raspberry pi, if that changes anything

**Comment by lamski** [comment]
- Is the error coming from flexget or from transmission?
- Is transmission installed correctly i.e. can you access the web interface and add torrents directly?
- Maybe you forgot to install `transmission-rpc`? Run `pip install --upgrade transmission-rpc` to install it.

**Comment by gazpachoking** [comment, marked as answer]
Are you trying to use the convert_magnet plugin? Transmission itself doesn't use libtorrent, but that plugin does.

**Comment by diddiman** [comment]
Sorry, forgot this thread.
But yes, I realized it was convert_magnet plugin. Ended up just skipping it, since I didn't really need it anyway. Thanks for your time though!!
